scss/_variables.scss

```diff
@@ -1 +1,44 @@
-$color-primary: #fff000;
+$color-primary: #FAFAFA;
+$grey:#3d4148;
+$container-width:1440px;
+%centering{
+    display: flex;
+    justify-content: center;
+}
+%logo{
+    img{
+    display: flex;
+    justify-content: center;
+    margin: 20px auto;
+  width: 105px;
+  height: 88px;
+}
+p{
+    display: flex;
+    justify-content: center;
+    font-size: 32px;
+    color:#FAFAFA;
+}
+}
+%headline{
+    width:541px;
+    height: 100px;
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    margin:160px auto;
+    h2{
```

Comment:
It's better to use classes instead of tags. It related to all elements.

Reply:
I will ask about it tomorrow. Difficult to understand via documentation(